docs/src/components/pricing/FAQ.tsx

```diff
@@ -92,15 +90,12 @@ const faqData = [
     summary: 'Am I allowed to use the product after the update entitlement expires?',
     detail: (
       <React.Fragment>
-        Yes. The license is perpetual, so you are allowed to continue using the product even after
-        the entitlement expires. However, you will be using the last product version released before
-        this time. You will lose access to subsequently released functionality and updates, as well
-        as technical support.
+        Yes. You can continue to use the product in production environments after the entitlement
+        expires. But you will need to renew your license to continue development, update for new
```

Comment:
Would this make more sense? Not sure, because for each payment a new license key, so there is a notion of renewal, but "renewal" suggests that it's optional. Not sure.

```suggestion
        expires. But you will need to keep your subscription active to continue development, update for new
```